template/makeCpp.js

```diff
@@ -688,7 +688,7 @@ function getResultActions(tabbing, apiCall, isInstanceApi) {
         return tabbing + "if (outResult.EntityToken.Len() > 0)\n"
             + tabbing + "    " + getContextContainer(isInstanceApi, true) + "SetEntityToken(outResult.EntityToken);\n";
     if (apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
-        return tabbing + "if (outResult.EntityToken.Len() > 0 && outResult.EntityToken->EntityToken.Len() > 0)\n"
+        return tabbing + "if (outResult.EntityToken != nullptr && outResult.EntityToken->EntityToken.Len() > 0 && outResult.EntityToken->EntityToken.Len() > 0)\n"
```

Comment:
There are 2 clauses with "outResult.EntityToken->EntityToken.Len() > 0", we need to remove 1